tests/ci/README.md

```diff
@@ -0,0 +1,135 @@
+CI pipelines.
+
+It also allow us to run the pipeline locally, in a virtual machine or remote
+server, which makes the developing and debugging much easier.
+
+## Structure
+
+The tests and provisioning scripts are located under the `tests/ci/` directory:
+
+
+```shell
+tests/
+└── ci
+    ├── clean_up.sh
+    ├── create-commit.sh
+    ├── files
+    │   ├── blueprint.toml
+    │   ├── centos-stream-8.json
+    │   ├── copr_neptune_tmpl.toml
+    │   ├── integration-net.xml
+    │   ├── ks_tmpl.cfg
+    │   ├── rhel-8-4-0-os-release
+    │   └── rhel-8-4-0-rh-release
+    ├── image-build.fmf
+    ├── install-vm.sh
+    ├── README.md
+    ├── setup.sh
+    └── test-vm.sh
+```
+
+The file that describes the CI pipeline is `image-build.fmf`. There is the
+definition of the steps, the order, how to prepare the environment, how to
+clean up after it finishes and some environment variables defined.
+
+All the templates and files to be copied at the directory `tests/ci/files`.
+
+## How to run locally the CI pipelines
+
+**IMPORTANT**: The tests will install and change some things in the host,
+so it's not recommended to run it in your own machine or production host.
+
+To run the tests locally you need to have `tmt` installed. The command `tmt`
+can run the provisioning and the tests in the same machine, in another
+(via `ssh`) or in a virtual machine. This last option is the default option.
+
+### In a fresh virtual machine
+
+You can run the tests inside a virtual machine (VM), by running the following:
+
+```shell
+tmt run -a -vvv \
+    provision --how virtual \
+    --image http://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20210603.0.x86_64.qcow2 \
+    --memory 8192 \
+    --disk 60
+```
+
+The `provision --how virtual` makes sure it runs in a VM. But the default
+option is a Fedora VM. To force it to use **CentOS Stream 8** you'll need
+to indicate a Centos Stream 8 image.
+
+To be able to run some of the steps, you'll need at least 8 GB of RAM and
+around 60 Gb of disk. The previous command takes care of all those limits.
+
+**NOTE**: For using the `virtual` provisioner you should install the `tmt-all`
+package.
+
+### In a CentOS 8 or CentOS Stream 8 machine
+
+It also can run in the same machine, but it needs to be CentOS 8 or CentOS
+Stream 8. It doesn't matter if the machine is virtual (like with Vagrant or
+libvirt), or if it's a bare-metal, but it needs to be able to run virtual
+machines inside.
+
+The steps to run the tests are:
+
+1. Connect to the machine you want to use.
+1. Make sure you have `tmt` and `git` installed (`dnf install -y tmt git`)
+1. Clone this repo: `git clone https://github.com/osbuild/automotive-ci.git`
+1. Run `tmt` locally with the following command:
+
+```shell
+tmt run -a -vvv provision --how local
+```
+
+### In a remote machine
+
+You'd run in a remote machine. You just need to have `tmt` installed locally
```

Comment:
maybe it'd be clearer to say "In order to run it on a remote machine you just..."

Reply:
I like it! Thanks!